enzyme/test/Enzyme/ForwardMode/maximum.ll

```diff
@@ -0,0 +1,29 @@
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -S | FileCheck %s
```

Comment:
The old pass manager line needs to be version gated to be < 16.

Separately if this intrinsic is only LLVM15+, you also need to version gate all tests to be 15+.